root/edit/edit_header.tt

```diff
@@ -31,7 +31,12 @@
                     </td>
                     <td class="vote-count">
                         [%~ IF edit.latest_vote_for_editor(c.user.id) || c.user.id == edit.editor_id || !edit.is_open ~%]
-                            <div>[%- vote_tally(edit) -%]</div>
+                            <div>
+                                [%~ IF !c.user_exists ~%]
+                                    <strong>[%- l('Vote tally: ') -%]</strong>
```

Comment:
This adds a new string, since the one in root/edit/index.tt doesn't have a space after the `:`

Actually, we also have a string without the colon in root/edit/search_macros.tt, so perhaps we should replace both this and the one in root/edit/index.tt to use that + add_colon.